app/views/appliances/show.html.haml

```diff
@@ -61,5 +64,81 @@
           = comment.body
 
 
-.w-full.mt-8
-  = render 'appliance_comments/compact_form', appliance_comment: @new_appliance_comment
+%link{:href => "https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css", :rel => "stylesheet"}/
+:css
```

Comment:
We should move this css in `app/assets/stylesheets/appliances.scss`.
The asset pipelin is not used right now so this may not be available.
We need to link to these css files in `app/views/layouts/application.html.haml` with
`  = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'`.
Other improvment on asset handling could go in another PR.